### DISCUSSION TITLE
When using GPU, does Milvus support a compute capability of 8.6

### DISCUSSION THREAD

**Discussion by luckercs** [milvus-io/milvus#27748] [Q&A and General discussion]
Our GPU card has a compute capability of 8.6.  We want to run milvus on the GPU card.

However, upon reviewing the official documentation, it was found that only 6.1, 7.0, 7.5, or 8.0 are supported

When using GPU, does Milvus support a compute capability of 8.6?

**Comment by yhmo** [comment]
Not supported in current versions. We have done the major work for supporting 8.6 in the index engine. The near future version(maybe 2.3.3) will support it.
